virttest/sriov.py

```diff
@@ -0,0 +1,110 @@
+"""
+SRIOV related utility functions
+"""
+
+import logging
+
+from avocado.core import exceptions
+
+from virttest import utils_misc
+from virttest import utils_net
+from virttest import remote
+from virttest.staging import service
+
+
+def find_pf(driver, session=None):
+    """
+    Check if available PF exists to test
+
+    :param session: The session object to the host
+    :raise: test.cancel when no pf found
+    :return: (interface name, pf device)
+    """
+    driver_dir = "/sys/bus/pci/drivers/%s" % driver
+    cmd = "ls -d %s/000*" % driver_dir
+    s_, output = utils_misc.cmd_status_output(cmd, shell=True,
+                                              ignore_status=False,
+                                              session=session)
+    runner = None
+    if session:
+        runner = session.cmd
+    pf_tmp = ()
+    pf_devices = output.splitlines()
+    for pf in pf_devices:
+        cmd = "ls %s/net" % pf
+        s_, tpm_iface_name = utils_misc.cmd_status_output(cmd, shell=True,
+                                                          ignore_status=False,
+                                                          session=session)
+
+        if (utils_net.get_net_if_operstate(
+           tpm_iface_name.strip(), runner=runner) == "up"):
+            pf_tmp = (tpm_iface_name.strip(), pf.strip().split('/')[-1])
+            break
+
+    if not pf_tmp:
+        raise exceptions.TestError("NO available pf found.")
+    return pf_tmp
+
+
+def set_vf(pci_addr, vf_no=4, session=None):
+    """
+    Enable VFs for PF
+
+    :param pci_addr: The pci address
+    :param vf_no: The value to be set
+    :param session: The session object to the host
+    :return: True if successful
+    """
+    logging.debug("pci_addr is %s", pci_addr)
+    cmd = "echo %s > %s/sriov_numvfs" % (vf_no, pci_addr)
+    s, o = utils_misc.cmd_status_output(cmd, shell=True,
+                                        verbose=True, session=session)
+    if not s:
```

Comment:
generally , we need keep one function or method to have one return value in whatever situation if you want them return value. 
So suggest change to :
return not s